tpcx_bb/benchmark_runner/draco/README.md

```diff
@@ -0,0 +1,29 @@
+This folder is intended for dev/testing/benchmarking our tpcx-bb queries in Slurm clusters.
+
+The Docker images used are date tagged but not intended to signify "stable" images.
+
+Setup:
+1. Somewhere you have Docker permissions, build image:
+```bash
+cd $TPCX_BB/tpcx_bb/benchmark_runner/draco
+docker build -t draco .
+# update image id
+docker tag bb38976d03cf dockerhub_username/draco:8_24_0
```

Comment:
`bb38976d03cf` is this a random image_id? Maybe replace with {image_tag} or something similar?